internal/storage/storage.go

```diff
@@ -336,3 +345,15 @@ func (s *Storage) mustGet(table, index string, args ...interface{}) memdb.Result
 
 	return it
 }
+
+func (s *Storage) first(table, index string, args ...interface{}) interface{} {
```

Comment:
I am writing this comment to let you know I don't understand this bit. But I'm assuming it does something important :sweat_smile: